packages/components/select/__tests__/select.test.tsx

```diff
@@ -1179,6 +1179,181 @@ describe("validation", () => {
     user = userEvent.setup();
   });
 
+  describe("validationBehavior=native", () => {
+    it("supports isRequired", async () => {
+      const {getByTestId} = render(
+        <Form data-testid="form">
+          <Select
+            isRequired
+            data-testid="select"
+            label="Test"
+            name="select"
+            validationBehavior="native"
+          >
+            <SelectItem key="one">One</SelectItem>
+            <SelectItem key="two">Two</SelectItem>
+            <SelectItem key="three">Three</SelectItem>
+          </Select>
+        </Form>,
+      );
+
+      const select = getByTestId("select");
+      const input = document.querySelector<HTMLSelectElement>("[name=select]");
```

Comment:
Select is a `<button>` and input is a `<select>`, please consider renaming the variables in the these tests